leapp/repository/__init__.py

```diff
@@ -189,6 +190,7 @@ def mapped_actor_data(data):
             })
             return data
 
+        # TODO: Do we need to add configs here as well?
```

Comment:
This is a good question, I think we need to decide and have a discussion on the position of configs in the "hierarchy" of leapp (how tightly is a config definition coupled to given actor, what is its scope...). Currently I am thinking about Configs having similar status as a Model. Meaning it is an entity that has the whole repository as a scope and is not bound to a specific actor. From that point of view I think the information would fit here.

Could you maybe explain what is your view and thought process?

Reply:
I may be dragging in an unrelated discussion: I remember that we were figuring out how we could make sure that changes to a config schema that was shared amongst several different Actors would be caught and error before an upgrade started.  IIRC, the strategy we decided upon was for the config schema to be copied into every Actor that wanted to use it.  Then the framework could detect if these opies got out of sync and error out (forcing the user to fix the code before continuing).

Would that mean we do not want configs added here?

Reply:
I'll be honest I am a bit lost about what we agreed on at this point :joy: However, in such a case I agree that it should _not_ be added here and the information should be present only in conjunction with actors just as dialogs are